lib/scoring_bowling.rb

```diff
@@ -0,0 +1,67 @@
+class ScoringBowling
+  MAX_FRAMES = 10
+  MAX_PINS = 10
+  attr_reader :rounds, :frame, :score
+
+  def initialize
+    @rounds = []
+    @frame = 0
+    @score = 0
+  end
+
+  def add_bowl(*bowls)
+    raise 'Game Over, no more scores can be added' if @rounds.length == MAX_FRAMES
+    
+    @rounds << { frame: frame_count, bowls: [*bowls] }
```

Comment:
It would be good if there was a way to add scores for individual rolls rather than entire frames.